Makefile

```diff
@@ -6,16 +6,16 @@ release_drivers=$(foreach d, $(drivers), release/$(d))
 
 all: $(drivers)
 
-$(drivers): FORCE
+$(drivers): sht-common/git_version.c FORCE
 	cd $@ && $(MAKE) $(MFLAGS)
 
 sht-common/git_version.c: FORCE
-	git describe --always --dirty | \
+	[ -d ".git" ] && git describe --always --dirty | \
 		awk 'BEGIN \
 		{print "/* THIS FILE IS AUTOGENERATED */"} \
 		{print "#include \"git_version.h\""} \
 		{print "const char * SHT_DRV_VERSION_STR = \"" $$0"\";"} \
-		END {}' > $@
+		END {}' > $@ || echo "Can't update version, not a git repository"
 
 
 $(release_drivers): sht-common/git_version.c
```

Comment:
So if I understand correctly previously the `git_version.c` was only generated when building the release_driver?

Reply:
Correct

Reply:
but it would linger around if not deleted manually, which is probably why I didn't notice.. let me clean it out as well